src/presentation/http/http-server.ts

```diff
@@ -51,6 +52,11 @@ export default class HttpServer implements API {
      */
     this.server.register(NoteRouter, { prefix: '/note',
       noteService: domainServices.noteService });
+    /**
+     * Register cors
+     */
+    this.server.register(cors, { origin: '*' });
```

Comment:
seems dangerous. Maybe it should be set on the infrastructure level? Or use "*" only for DEV env

Reply:
done